binding/java/test/ai/picovoice/cheetah/CheetahPerformanceTest.java

```diff
@@ -37,12 +37,12 @@ void initPerformance() throws Exception {
         long[] perfResults = new long[numTestIterations];
         for (int i = 0; i < numTestIterations; i++) {
             long before = System.nanoTime();
-            Cheetah cheetah = new Cheetah(
-                    accessKey,
-                    Utils.getPackagedLibraryPath(),
-                    Utils.getPackagedModelPath(),
-                    1
-            );
+            Cheetah cheetah = new Cheetah.Builder()
```

Comment:
These props have the same incorrect average math. Need to iterate for `numTestIterations + 1`